controllers/login.js

```diff
@@ -0,0 +1,26 @@
+const ModelUser = require('../models/user')
+const Guid = require('guid')
+
+module.exports.login = async (req, res) => {
+  try {
+    const email = req.body.email
+    const password = req.body.password
```

Comment:
use a destructor for this two lines of code `const {email, password} = req.body`